### PR TITLE
Town's crud system and design

### PR DESCRIPTION
**Pull request by oramasangel**
# Description

 - I've created the structure for crud's system
 - I've designed the views
 - I've implemented calling the township names to display them in the id fields of the towns view
 
 Examples:
 
 # Testing
 
 Example:
  
  ## Screenshots

### CODE REVIEW

File: local_sales/test/controllers/towns_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (jairolan) [inline local_sales/test/controllers/towns_controller_test.rb:1]: If you don't use this file, remove it

File: local_sales/app/helpers/towns_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module TownsHelper
+end
```
Comment (juliocabrera820) [inline local_sales/app/helpers/towns_helper.rb:2]: remove file

File: local_sales/app/views/towns/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        </tr>
+      </thead>
+      <tbody>
+        <% @towns.each do |town| %>
+        <tr>
+          <td class="py-4 px-1"><%= town.name %></td>
+          <td class="py-4 px-1"><%= @towns.find(town.id).township.name%></td>
+          <td class="py-4 lg:pl-2 pl-0"><%= link_to image_tag("edit_icon.svg"), edit_town_path(town) %></td>
+        </tr>
+        <% end %>
```
Comment (juliocabrera820) [inline local_sales/app/views/towns/index.html.erb:27]: add indentation to identify the row

File: local_sales/test/controllers/towns_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+class TownsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (juliocabrera820) [inline local_sales/test/controllers/towns_controller_test.rb:7]: remove file

**Review by juliocabrera820 — COMMENTED** [review]
Remove unneeded files and keep your indentation

File: local_sales/app/controllers/towns_controller.rb [hunk 5]
```diff
@@ -0,0 +1,52 @@
+class TownsController < ApplicationController
+    before_action :set_town, only: %i[edit update]
```
Comment (juliocabrera820) [inline local_sales/app/controllers/towns_controller.rb:2]: keep and indentation of 2 spaces

**Comment by brayandiazc** [comment]
Ok

**Review by brayandiazc — APPROVED** [review]
LGTM

**Review by juliocabrera820 — APPROVED** [review]
LGTM 👍🏼

File: local_sales/app/controllers/towns_controller.rb [hunk 6]
```diff
@@ -0,0 +1,52 @@
+class TownsController < ApplicationController
+  before_action :set_town, only: %i[edit update]
+
+  # GET /states
+  def index
+    @town = Town.all
+    @townships = Township.all
```
Comment (dannharo) [inline local_sales/app/controllers/towns_controller.rb:7]: I see that you are not using this variable, Is it needed?

Comment (oramasangel) [reply local_sales/app/controllers/towns_controller.rb:7]: i thought that i needed to call the name of the township, i'll fix in a new issue for bugs/unnecessary things

**Review by jairolan — APPROVED** [review]
Nice Job

**Pull request merged**